### PR TITLE
[BH-000] update harmony changelog 2.2.1

### PR DESCRIPTION
**Pull request by ghost**
**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: image/system_a/data/lang/Svenska.json
Comment (lukasmas) [inline image/system_a/data/lang/Svenska.json:1]: Do not remove this file in master branch

Comment (Lefucjusz) [reply image/system_a/data/lang/Svenska.json:1]: True, this will remove the support for Swedish for Pure!

File: harmony_changelog.md [hunk 1]
```diff
@@ -3,21 +3,28 @@
...
+* Fixed not disappearing snooze icon when an alarm is deactivated
+* Fixed incorrect message after new alarm setting in some scenarios
+* Fixed the possibility of OS crash during update package size check
+* Fixed errors that occur when uploading a music file with graphics in the metadata via the Center.
+
+### Changed / Improved
+* Improved the display time of the "turn off" window.
+* Hardware faults during device startup now result in shutdown and a more user friendly message
 * Discharged device screen changed
 * Disabled USB MTP protocol
```
Comment (Lefucjusz) [inline harmony_changelog.md:29]: Has this been released in 2.2.1? I can't see this commit in release/2.2.1-bell.

**Review by lukasmas — APPROVED** [review]

**Review by lukasmas — CHANGES_REQUESTED** [review]
Use just single commit, this one with changelog update

File: harmony_changelog.md [hunk 2]
```diff
@@ -3,21 +3,28 @@
...
-* Fixed not disappearing snooze icon when an alarm is deactivated
-* Fixed incorrect message after new alarm setting in some scenarios
 * Fixed frequency lock during user activity
-* Fixed possibility of OS crash during update package size check
-* Fixed incorrect calculation of requested CPU frequency
-* Fixed CPU frequency setting when dumping logs to a file
+
 
 ### Added
 * Files not fully transferred via Center will be now removed when USB cable is unplugged
```
Comment (Lefucjusz) [inline harmony_changelog.md:10]: This was released in 2.2.1.

**Review by lukasmas — APPROVED** [review]
Add proper commit name and it should be good to go :)

**Review by dawidwojtas-mudita — APPROVED** [review]

**Pull request merged**